site/app/views/HomePageView.php

```diff
@@ -2,16 +2,20 @@
 
 namespace app\views;
 
+//check if needed
+//use app\libraries\Access;
```

Comment:
The changes in this file should be reviewed.  I think a lot of them are probably unnecessary/unintended.  Also: see prior comment about not needing the CSRF token since it's already on every page.